code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -224,9 +224,12 @@
 		pump_reload(M)
 	else
 		pump_unload(M)
-	bolt_open = !bolt_open
+	update_icon(UPDATE_ICON_STATE)
 	return 1
```

Comment:
I am not sure you meant to remove this line? bolt_open never gets set to true

Reply:
oh damn